docs/api/stickers.rst

```diff
@@ -0,0 +1,108 @@
+.. SPDX-License-Identifier: MIT
+
+.. currentmodule:: disnake
+
+Stickers
+========
+
+This section documents everything related to stickers.
+
+Classes
+-------
+
+Sticker
+~~~~~~~
+
+.. attributetable:: Sticker
+
+.. autoclass:: Sticker()
+    :members:
+    :inherited-members:
+
+StandardSticker
+~~~~~~~~~~~~~~~~
+
+.. attributetable:: StandardSticker
+
+.. autoclass:: StandardSticker()
+    :members:
+    :inherited-members:
+
+GuildSticker
+~~~~~~~~~~~~~
+
+.. attributetable:: GuildSticker
+
+.. autoclass:: GuildSticker()
+    :members:
+    :inherited-members:
+
+StickerItem
+~~~~~~~~~~~~~
+
+.. attributetable:: StickerItem
+
+.. autoclass:: StickerItem()
+    :members:
+    :inherited-members:
+
+Data Classes
+------------
+
+StickerPack
+~~~~~~~~~~~~~
+
+.. attributetable:: StickerPack
+
+.. autoclass:: StickerPack()
+    :members:
+
+Enumerations
+------------
+
+StickerType
+~~~~~~~~~~~
+
+.. class:: StickerType
+
+    |discord_enum|
+
+    Represents the type of sticker.
+
+    .. versionadded:: 2.0
+
+    .. attribute:: standard
+
+        Represents a standard sticker that all Nitro users can use.
+
+    .. attribute:: guild
+
+        Represents a custom sticker created in a guild.
+
+StickerFormatType
+~~~~~~~~~~~~~~~~~
+
+.. class:: StickerFormatType
+
+    |discord_enum|
+
+    Represents the type of sticker images.
+
+    .. versionadded:: 1.6
+
+    .. attribute:: png
+
+        Represents a sticker with a png image.
+
+    .. attribute:: apng
+
+        Represents a sticker with an apng image.
+
+    .. attribute:: lottie
+
+        Represents a sticker with a lottie image.
+
+Events
+------
+
+Check out the :ref:`related events <related_events_sticker>`!
```

Comment:
I feel like each of these should list the events that are related and link to their proper defintions.

Reply:
You mean something like this?
```rst
Voice
=====
...

Events
------

- :ref:`on_voice_state_update(member, before, after) <disnake_api_events_on_voice_state_update>`
```

Reply:
Yeah, that sounds good.

Reply:
Something like this, which I think would work.
```rst
Voice
=====
...

Events
------

- :ref:`on_voice_state_update(member, before, after) <disnake.on_voice_state_update>`
- (all events...)

*See all `related events <related_events_voice>`.*
```